### PR TITLE
Fixing OIE Migrate style issues

### PR DESCRIPTION
**Pull request by jakubvul-okta**
Fixing style issues

## Description:
- **What's changed?** Fixed doc style issues
- **Is this PR related to a Monolith release?** No

### Resolves:

* N/A

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 1]
```diff
@@ -10,19 +10,19 @@ layout: Guides
 
 To enable Okta Identity Engine, please reach out to your account manager. If you do not have an account manager, please reach out to <oie@okta.com> for more information.
 
-## Enable `interaction code` grant
+## Enable interaction code grant
 
 Once the Okta Identity Engine feature is enabled for your org, it should become active for Okta-hosted sign-in flows that do not involve an OAuth application. Enabling the `interaction_code` grant type will allow OAuth applications to use the Okta Identity Engine.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:15]:
```suggestion
After the Okta Identity Engine feature is enabled for your org, it should become active for Okta-hosted sign-in flows that don't involve an OAuth application. Enabling the `interaction_code` grant type allows OAuth applications to use the Okta Identity Engine.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ layout: Guides
...
 
-### Enable `interaction code` grant on an authorization server
+### Enable interaction code grant on an authorization server
 
 1. Navigate to `Security > API > Authorization Servers` in the Admin UI
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:19]:
```suggestion
1. Navigate to **Security > API > Authorization Servers** in the Admin Console.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 3]
```diff
@@ -10,19 +10,19 @@ layout: Guides
...
 2. Select an authorization server and click the edit icon.
 3. Click on `Access Policies` tab.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:21]:
```suggestion
3. Click the **Access Policies** tab.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 4]
```diff
@@ -10,19 +10,19 @@ layout: Guides
...
 4. Edit `Default Policy Rule`.
 5. In the section `IF Grant type is`, click the `Interaction Code` checkbox.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:23]:
```suggestion
5. In the **IF Grant type is** section, click the **Interaction Code** check box.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 5]
```diff
@@ -10,19 +10,19 @@ layout: Guides
...
 
-### Enable `interaction code` grant on an application
+### Enable interaction code grant on an application
 
 1. Navigate to `Applications > Applications` in the Admin UI
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:27]:
```suggestion
1. Navigate to **Applications > Applications** in the Admin Console.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 6]
```diff
@@ -10,19 +10,19 @@ layout: Guides
...
 2. Click an application in the list
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:28]:
```suggestion
2. Click an application in the list.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 7]
```diff
@@ -40,10 +40,10 @@ For most authentication flows that involve redirecting to Okta, there should be
 
 For most users of the [custom domain](/docs/guides/custom-url-domain/overview/) feature, there are no other changes needed. The default template will detect and use the Okta Identity Engine automatically.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:41]:
```suggestion
For most users of the [custom domain](/docs/guides/custom-url-domain/overview/) feature, there are no other changes needed. The default template detects and uses the Okta Identity Engine automatically.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 8]
```diff
@@ -40,10 +40,10 @@ For most authentication flows that involve redirecting to Okta, there should be
...
 
-However if you have [modified the template](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/) in certain ways, such as to perform redirects or set cookies, these modificiations may not be compatible with the Okta Identity Engine. In particular, these methods and objects will not work with OIE:
+However if you have [modified the template](/docs/guides/style-the-widget/style-okta-hosted/) in certain ways, such as to perform redirects or set cookies, these modificiations may not be compatible with the Okta Identity Engine. In particular, these methods and objects will not work with OIE:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:43]:
```suggestion
However, if you have [modified the template](/docs/guides/style-the-widget/style-okta-hosted/) in certain ways (such as to perform redirects or set cookies), these modifications may not be compatible with the Okta Identity Engine. In particular, these methods and objects won't work with OIE:
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 9]
```diff
@@ -40,10 +40,10 @@ For most authentication flows that involve redirecting to Okta, there should be
...
 
-- setCookieAndRedirect
-- sessionToken
+- `setCookieAndRedirect`
+- `sessionToken`
 
 In the default template, the `success` callback from `renderEl` is being handled by `OktaUtil.completeLogin`. This method correctly handles the authentication flow and we recommend using it.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:48]:
```suggestion
In the default template, the `success` callback from `renderEl` is being handled by `OktaUtil.completeLogin`. This method correctly handles the authentication flow, and we recommend using it.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 10]
```diff
@@ -90,11 +90,11 @@ For reference, here is the default template:
 </html>
 ```
 
-### Embedded sign-in widget
+### Embedded Sign-In Widget
 
-> **Note:** "Embedded" means the widget is included directly in your application via NPM module or script tag. @okta/okta-signin-widget version 5.2.0 or above is needed to enable the interaction code flow.
+> **Note:** "Embedded" means the Sign-In Widget is included directly in your application via npm module or script tag. `@okta/okta-signin-widget` version 5.2.0 or above is needed to enable the interaction code flow.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:95]:
```suggestion
> **Note:** "Embedded" means the Sign-In Widget is included directly in your application through npm module or script tag. The`@okta/okta-signin-widget` version 5.2.0 or above is needed to enable the interaction code flow.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 11]
```diff
@@ -90,11 +90,11 @@ For reference, here is the default template:
...
 
-Set the option `useInteractionCodeFlow` to `true` on the object passed to the Sign-In Widget constructor. This will enable Okta Identity Engine for the widget. Both the authorization server and the application must have the [interaction code](#enable-interaction-code-grant) grant type enabled.
+Set the option `useInteractionCodeFlow` to `true` on the object passed to the Sign-In Widget constructor. This will enable Okta Identity Engine for the Widget. Both the authorization server and the application must have the [interaction code](#enable-interaction-code-grant) grant type enabled.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:97]:
```suggestion
Set the option `useInteractionCodeFlow` to `true` on the object passed to the Sign-In Widget constructor. This enables Okta Identity Engine for the Widget. Both the authorization server and the application must have the [interaction code](#enable-interaction-code-grant) grant type enabled.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 12]
```diff
@@ -10,19 +10,19 @@ layout: Guides
...
 
 Once the Okta Identity Engine feature is enabled for your org, it should become active for Okta-hosted sign-in flows that do not involve an OAuth application. Enabling the `interaction_code` grant type will allow OAuth applications to use the Okta Identity Engine.
 
-### Enable `interaction code` grant on an authorization server
+### Enable interaction code grant on an authorization server
 
 1. Navigate to `Security > API > Authorization Servers` in the Admin UI
 2. Select an authorization server and click the edit icon.
 3. Click on `Access Policies` tab.
 4. Edit `Default Policy Rule`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:22]:
```suggestion
4. Edit the **Default Policy Rule**.
```

**Review by susanharper-okta — COMMENTED** [review]
couple things

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 13]
```diff
@@ -10,22 +10,22 @@ layout: Guides
...
+5. In the **IF Grant type is** section, click the **Interaction Code** check box.
 
-### Enable `interaction code` grant on an application
+### Enable interaction code grant on an application
 
-1. Navigate to `Applications > Applications` in the Admin UI
-2. Click an application in the list
+1. Navigate to **Applications > Applications** in the Admin Console.
+2. Click an application in the list.
 3. Click the `General` tab on the application detail
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:29]:
```suggestion
3. Click the **General**` tab on the application detail.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 14]
```diff
@@ -10,22 +10,22 @@ layout: Guides
...
 4. Click edit on the `General Settings` panel
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:30]:
```suggestion
4. Click edit on the **General Settings** panel.
```

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 15]
```diff
@@ -10,22 +10,22 @@ layout: Guides
...
 5. In the section `Allowed grant types`, click the `Interaction Code` checkbox.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:31]:
```suggestion
5. In the **Allowed grant types** section, click the **Interaction Code** check box.
```

**Review by susanharper-okta — APPROVED** [review]
somehow I missed a few - approving to not hold you up.

File: packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md [hunk 16]
```diff
@@ -10,25 +10,25 @@ layout: Guides
...
+### Enable interaction code grant on an application
 
-1. Navigate to `Applications > Applications` in the Admin UI
-2. Click an application in the list
-3. Click the `General` tab on the application detail
-4. Click edit on the `General Settings` panel
-5. In the section `Allowed grant types`, click the `Interaction Code` checkbox.
+1. Navigate to **Applications > Applications** in the Admin Console.
+2. Click an application in the list.
+3. Click the **General**` tab on the application detail.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md:29]:
```suggestion
3. Click the **General** tab on the application detail.
```

**Pull request merged**